ern-local-cli/src/lib/utils.js

```diff
@@ -504,32 +504,70 @@ async function launchAndroidRunner (pathToAndroidRunner: string) {
 }
 
 async function launchIosRunner (pathToIosRunner: string) {
-  const iPhoneDevice = await ios.askUserToSelectAniPhoneDevice()
+  const iosDevices = ios.getiPhoneRealDevices()
+  if (iosDevices && iosDevices.length > 0) {
+    log.debug(`Will launch ErnRunner on a real device`)
+    launchOnDevice(pathToIosRunner, iosDevices)
+  } else {
+    launchOnSimulator(pathToIosRunner)
+  }
+}
+
+async function launchOnDevice (pathToIosRunner: string, devices) {
+  const iPhoneDevice = await ios.askUserToSelectAniPhoneDevice(devices)
+  shell.cd(pathToIosRunner)
+  const spinner = ora(`Waiting for device to boot`).start()
```

Comment:
minor: use single quote instead of backticks for pure string (non template litteral)